Feisty/Login/ViewController.swift

```diff
@@ -41,10 +51,11 @@ class ViewController: UIViewController {
     }
     
     viewModel.attemptLogin(with: username, and: password)
+    
   }
 
   func moveToMainMenu() {
-      performSegue(withIdentifier: "segToMain", sender: nil)
+    self.performSegue(withIdentifier: "segToMain", sender: nil)
```

Comment:
Do you have to use self here? If you don't, please remove it as it would not be according to SwiftLint standards.

Reply:
This is an issue that I have resolved in the current feature.